src/types/onyx/PolicyTag.ts

```diff
@@ -23,6 +23,9 @@ type PolicyTagList<T extends string = string> = Record<
 
         /** Nested tags */
         tags: PolicyTags;
+
+        /** Index by which the tag appears in the hierarchy of tags */
+        orderWeight: number;
```

Comment:
NAB: I'm confused why this looks like a change if https://github.com/Expensify/App/pull/37468 was merged 🤯

Reply:
Oh, I was branching off that first branch. If I pull I believe it will go away one sec.